pkgs/applications/graphics/gimp/default.nix

```diff
@@ -48,6 +47,8 @@
 , AppKit
 , Cocoa
 , gtk-mac-integration-gtk2
+, withPython ? false
+, python2 ? null
```

Comment:
Why do we need the ? null here? We are not going to drop the python2 package from the tree.

Reply:
We might in the future.

Reply:
Then we can still do this in the future or just remove the dead code.